<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>heart attack models</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-size:15px;
font-family:arial;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true">QPushButton:hover { color: red }</string>
       </property>
       <attribute name="title">
        <string>heart attack prediction</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="0" colspan="3">
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="spinAge">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>120</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Chest pain type</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="3">
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>0 = Typical Angina, 1 = Atypical Angina</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QRadioButton" name="t120">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>true</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroupbs</string>
               </attribute>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="man">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>man</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroupSex</string>
               </attribute>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QSpinBox" name="spinBp">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_7">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Cholestoral in mg/dl</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0" colspan="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Resting blood pressure (in Hg mm)</string>
               </property>
              </widget>
             </item>
             <item row="8" column="2">
              <widget class="QSpinBox" name="spinChol">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>600</number>
               </property>
              </widget>
             </item>
             <item row="9" column="2">
              <widget class="QRadioButton" name="f120">
               <property name="text">
                <string>false</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroupbs</string>
               </attribute>
              </widget>
             </item>
             <item row="5" column="0" colspan="3">
              <widget class="QLabel" name="label_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>2 = Non-anginal Pain, 3 = Asymptomatic </string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_8">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>fasting blood sugar &gt; 120 mg/dl</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QRadioButton" name="woman">
               <property name="text">
                <string>woman</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroupSex</string>
               </attribute>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>sex</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>age</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="3">
              <widget class="Line" name="line">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QSpinBox" name="spinCP">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>3</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="9" column="2">
              <widget class="QSpinBox" name="spinBoxThall">
               <property name="maximum">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="3">
              <widget class="Line" name="line_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QSpinBox" name="spinHeartRate">
               <property name="maximum">
                <number>250</number>
               </property>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QSpinBox" name="spinSlope">
               <property name="maximum">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Resting electrocardiographic results</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Exercise induced angina</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QDoubleSpinBox" name="doubleSpinBox">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Thalium Stress Test result</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QRadioButton" name="yes">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>yes</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroupexng</string>
               </attribute>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Number of major vessels</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="spinRestecg">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>slope</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QLabel" name="label_11">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>2 = Left ventricular hypertrophy</string>
               </property>
              </widget>
             </item>
             <item row="8" column="2">
              <widget class="QSpinBox" name="spinBoxCaa">
               <property name="maximum">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="label_10">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>0 = Normal, 1 = ST-T wave abnormality</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Maximum heart rate achieved</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Previous peak </string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QRadioButton" name="no">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>no</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroupexng</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>PREDICT NOW</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupSex"/>
  <buttongroup name="buttonGroupbs"/>
  <buttongroup name="buttonGroupexng"/>
 </buttongroups>
</ui>
